<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>930</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>532</width>
      <height>781</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="titleLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="genreLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="genreComboBox">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>-- Select Genre --</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fantasy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biography</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Drama</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Romance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thriller</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Historical Fiction</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Science Fiction</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Action &amp; Adventure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="totalPagesLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total Pages</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="totalPagesSpinBox">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="lastPageLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Last Page Read</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="lastPageSpinBox">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="progressLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Progress</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QProgressBar" name="progressBar">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="ratingLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rating</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnStar1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>☆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStar2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>☆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStar3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>☆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStar4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>☆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStar5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>30</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>☆</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="notStartedCheckBox">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Not Started</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QCheckBox" name="readingCheckBox">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reading</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QCheckBox" name="finishedCheckBox">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Finished</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="reviewLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Note/Review</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QTextEdit" name="textEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="authorLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="authorLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>40</horstretch>
           <verstretch>40</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add Book</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>40</horstretch>
           <verstretch>40</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Update Book</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>40</horstretch>
           <verstretch>40</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Delete Book</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="bookTable">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>891</width>
      <height>741</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bookInfoLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Book's Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="bookListLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Book Collection</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>890</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Regia Puspa Amaranthi - F1D022156</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
